liquibase-core/src/main/java/liquibase/snapshot/jvm/JdbcSnapshotGenerator.java

```diff
@@ -160,6 +161,10 @@ protected String cleanNameFromDatabase(String objectName, Database database) {
             }
             
         }
+
+        if (returnList.size() == 0 && database instanceof SQLiteDatabase) {
```

Comment:
can we expand this to other DBs? I found Firebird need same fix as it also fails with `liquibase.exception.UnexpectedLiquibaseException: Found a null snapshotId for catalog null`

Reply:
changing if condition to include (database instanceof SQLiteDatabase || database instanceof FirebirdDatabase) helps with Firebird snapshot generation, but maybe this can be expanded further

Reply:
@nvoxland -- please see @KushnirykOleh's comment above and provide guidance.

Reply:
Yeah, it is probably safe to include for all database, not just sqlite and firebird. I updated the code to just be `if (returnlist.size() == 0)`